iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -625,82 +628,89 @@ Inspector.prototype._footnoteFactsHTML = function(fact) {
     return html;
 }
 
+Inspector.prototype._factSummaryHTML = function(fact, factset) {
+    var factHTML;
+    if (fact instanceof Fact) {
+        factHTML = $(require('../html/fact-details.html')); 
+        $('.std-label', factHTML).text(fact.getLabelOrName("std", true));
+        $('.documentation', factHTML).text(fact.getLabel("doc") || "");
+        $('tr.concept td', factHTML).text(fact.conceptName());
+        $('tr.period td', factHTML)
+            .text(fact.periodString());
+        if (fact.isNumeric()) {
+            $('tr.period td', factHTML).append(
+                $("<span></span>") 
+                    .addClass("analyse")
+                    .text("")
+                    .click(() => this.analyseDimension(fact, ["p"]))
+            );
+        }
+        this._updateEntityIdentifier(fact, factHTML);
+        this._updateValue(fact, false, factHTML);
+
+        var accuracyTD = $('tr.accuracy td', factHTML).empty().append(fact.readableAccuracy());
+        if (!fact.isNumeric() || fact.isNil()) {
+            accuracyTD.wrapInner("<i></i>");
+        }
+
+        $('#dimensions', factHTML).empty();
+        for (const aspect of fact.aspects()) {
+            if (!aspect.isTaxonomyDefined()) {
+                continue;
+            }
+            const h = $('<div class="dimension"></div>')
+                .text(aspect.label() || aspect.name())
+                .appendTo($('#dimensions', factHTML));
+            if (fact.isNumeric()) {
+                h.append(
+                    $("<span></span>") 
+                        .addClass("analyse")
+                        .text("")
+                        .on("click", () => this.analyseDimension(fact, [aspect.name()]))
+                )
+            }
+            const s = $('<div class="dimension-value"></div>')
+                .text(aspect.valueLabel())
+                .appendTo(h);
+            if (aspect.isNil()) {
+                s.wrapInner("<i></i>");
+            }
+        }
+        const nsURI = fact.concept().qname().namespace;
+        $('tr.taxonomy td', factHTML).text(this._report.taxonomyNameForURI(nsURI, nsURI));
+    }
+    else if (fact instanceof Footnote) {
```

Comment:
Pre-existing, but would it be better to use `item` naming (throughout) rather than `fact`?  `_itemSummaryHTML`, `ci` (current item?), etc.

Reply:
Have split into separate methods so that we can have a descriptive variable name in both branches, and renamed to item in the common code.